docs/source/technical_tutorials/working_with_r/access_aws_maap.ipynb

```diff
@@ -0,0 +1,1588 @@
+{
```

Comment:
<p>nit-pick: add space around <code>=</code>: <code>sep = "/"</code> (per the style guide I linked in another comment)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/471/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>